### ISSUE TITLE
Unable to use persistent valume from existing claim

### ISSUE THREAD

**Issue by justinwoodland** [Joxit/helm-charts#3]
I'm trying to set a registry volume with `persistentVolumeClaim` instead of an `emptyDir`, but setting values.yaml like:
```
registry:
  dataVolume:
    persistentVolumeClaim:
      claimName: "registry"
```
results in a rendered template like:
```
...
      volumes:
        - name: data
          emptyDir: {}
          persistentVolumeClaim:
            claimName: registry
...
```
The default values.yaml sets:
```
registry:
  dataVolume:
    emptyDir: {}
```
so it appears that the chart is merging the two objects instead of replacing it.

I'm not sure how to get around this.

**Comment by Joxit** [comment]
Hi, thank you for using my project and submitting an issue :smile: 

Issue confirmed, I will fix this ASAP

**Comment by Joxit** [comment]
Hi again, this may be fixed in 0.2.1 :rocket:
